app/controllers/application_controller.rb

```diff
@@ -45,4 +46,12 @@ def set_variant
       request.variant = :redesign
     end
   end
+
+  def update_recently_active
+    return unless current_user&.developer
+    return unless current_user.developer&.badge
+    return if current_user.developer.badge.recently_active?
+
+    UpdateDeveloperRecentlyActiveJob.perform_now(current_user.developer.id, true)
```

Comment:
I think we still need to save _when_ the developer was last active. This is now the only place we are updating the column, which means it will never be set to false.

Also, if we are going to use a job let's keep the logic there and pass in `current_user&.developer_id`. Then the job can check if the developer exists and if so set the timestamp on the badge.